Sources/PointFreeRouter/Routes.swift

```diff
@@ -64,6 +63,12 @@ public enum Route: Equatable {
     case requestInvite(EnterpriseAccount.Domain, EnterpriseRequestFormData)
   }
 
+  public enum EpisodeRoute: Equatable {
+    case index
+    case progress(param: Either<String, Episode.Id>, percent: Int)
+    case show(Either<String, Episode.Id>)
```

Comment:
the new route enum

Reply:
Could also change `EpisodeRoute` to `EpisodesRoute` and that way we could start grouping more specific routing for particular episodes:

```swift
case episode(Either<String, Episode.Id>, EpisodeRoute)
…
.episode(.left(slug), .show)
.episode(.left(slug), .progress(0.5)
```

Reply:
def down to do that, but how do you envision support `index` too? just a `case episodes` next to `case episode`?